bundle/Resources/views/default/content/embed_image.html.twig

```diff
@@ -0,0 +1,13 @@
+{% set image_field_identifier = ez_content_field_identifier_first_filled_image(content) %}
```

Comment:
This function name is new in eZ Platform v3. In v2 it was called `ez_first_filled_image_field_identifier`.

I have not found a sane way to implement compatibility between them :/

Reply:
Maybe `ng_content_field_identifier_first_filled_image`? 😂

Reply:
:D

Actually, now that you mention it, I remembered I did implement the exact same BC layer in Layouts & eZ Platform integration: https://github.com/netgen-layouts/layouts-ezplatform/commit/7d21b3655f7b2308535b644f3bf48f4029ffbe20

Reply:
I think I'll go the same way. One alternative would be to implement `function_exists`, but it does not seem the better approach.

Reply:
It turns out this one can be easily written in Twig, updated in https://github.com/netgen/ezplatform-site-api/pull/130/commits/b1f4bfb60b47a08b074680f8bc1ba76893c88fac